api/test/inet-grouping.test.js

```diff
@@ -126,3 +191,16 @@ const createRequestFromAddress = (remoteAddress, headers) => {
     headers
   })
 }
+
+const givenNetworkInfoUpdateHistory = async (pgClient, day, stationId) => {
+  await pgClient.query(`
+    INSERT INTO network_info_update_history (day, station_id)
+    VALUES ($1, $2)
+    ON CONFLICT (day, station_id) DO NOTHING
+  `, [day, stationId])
+}
+
+const getCurrentDate = async () => {
+  const { rows: [{ today }] } = await pgClient.query('SELECT now()::DATE::TEXT as today')
+  return today
+}
```

Comment:
I have seen this code in other tests files, what do you think about extracting it into a shared test helper and create a new file in the [/test-helpers](https://github.com/filecoin-station/spark-api/tree/main/test-helpers) directory?

Reply:
Makes sense! Due to more recent discussion, this is not needed anymore due to in-memory structure instead though